lib/cmsis_rtos_v1/cmsis_thread.c

```diff
@@ -49,7 +49,7 @@ osThreadId osThreadCreate(const osThreadDef_t *thread_def, void *arg)
 		 thread_def->stacksize <= CONFIG_CMSIS_THREAD_MAX_STACK_SIZE,
 		 "invalid stack size\n");
 
-	if (thread_def->instances == 0) {
+	if ((!thread_def) || (thread_def->instances == 0)) {
```

Comment:
Use thread_def == NULL to maintain uniformity

Reply:
Done